### PR TITLE
Option for rectangle domain fill opacity

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

> What was changed in this pull request?

- Added a menu option for rectangle domain fill opacity

> Why is it necessary?

Fixes #___

## Checklist

- [x] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [ ] Unit tests added or updated
- [ ] Documentation added or updated
- [ ] Example(s) added or updated
- [ ] Update schema.json if there are changes to the viewconf JSON structure format
- [ ] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/ArrowheadDomainsTrack.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ class ArrowheadDomainsTrack extends TiledPixiTrack {
         : 0.4,
     );
 
-    graphics.alpha = this.options.rectangleDomainOpacity || 0.5;
+    // graphics.alpha = this.options.rectangleDomainOpacity || 0.5;
```
Comment (flekschas) [inline app/scripts/ArrowheadDomainsTrack.js:252]: Why is this commented out here? Does the track now support it?

**Review by flekschas — COMMENTED** [review]
Looks good. Just don't know about the ArrowheadDomainsTrack change

File: app/scripts/ArrowheadDomainsTrack.js (hunk 1 above)
Comment (pkerpedjiev) [reply app/scripts/ArrowheadDomainsTrack.js:252]: Oversight 😬  Commented out for testing and forgot to return.

Comment (pkerpedjiev) [inline app/scripts/ArrowheadDomainsTrack.js:252]:
```suggestion
    graphics.alpha = this.options.rectangleDomainOpacity || 0.5;
```

**Pull request merged**